docs/Getting Started/Fedora/index.rst

```diff
@@ -0,0 +1,64 @@
+.. highlight:: sh
+
+Fedora
+======
+
+.. contents:: Table of Contents
+  :local:
+
+Installation
+------------
+
+If you want to use ZFS as your root filesystem, see the `Root on ZFS`_
+links below instead.
+
+Currently, due to Fedora's unstable fast-paced kernel ABI (kABI), only DKMS style packages can be provided at this time.
+As of right now, the following configurations are supported:
+  
+**Fedora Releases**: 30, 31, 32
```

Comment:
Fedora 30 does not seem to be supported upstream (i.e. by Fedora) any more. If that is the case, please remove it here and below.

Reply:
Yeah, I put that here because Fedora 30 was still in the wiki for Fedora

Reply:
Wait, should I remove it?